core/src/main/kotlin/com/example/core/dto/response/DeviceResponseDto.kt

```diff
@@ -0,0 +1,13 @@
+package com.example.core.dto.response
+
+import com.example.core.dto.DeviceStatusType
+
+data class DeviceResponseDto(
+    val name: String?,
+
+    val description: String?,
+
+    val type: String?,
+
+    val statusType: DeviceStatusType?,
```

Comment:
should these parameters be nullable?

Reply:
<img width="850" alt="Screenshot 2024-10-25 at 11 22 15" src="https://github.com/user-attachments/assets/9827b9a5-03c4-4a70-8356-7e669451c656">

Reply:
shtosh